src/workers/cdc/primary.ts

```diff
@@ -624,14 +624,6 @@ const onSourceChange = async (
   logger: FastifyBaseLogger,
   data: ChangeMessage<Source>,
 ) => {
-  logger.info(
-    {
-      op: data.payload.op,
-      before: data.payload.before,
-      after: data.payload.after,
-    },
-    'source change',
-  );
```

Comment:
Can keep this here till after debezium change, just to be sure.

Reply:
+1